.github/workflows/issue-management.yml

```diff
@@ -45,6 +45,12 @@ jobs:
                 milestone: milestone.number
               });
             }
+            await github.rest.issues.removeLabel({
```

Comment:
@samsmithnz This should probably be outside of the P1 IF condition, we want to do this on **any** issue that recieves labels really.

Reply:
S33 https://github.com/satisfactory-factories/application/issues/138 as an example